ci/check_diff.sh

```diff
@@ -89,10 +90,18 @@ function check_diff() {
         --unified=0 --no-index rustfmt_diff.txt feature_diff.txt 2>&1 | tail -n +6 | cut -c 2-
     )
 
+    # COPY diffs into output dir
+    mkdir $OUTPUT_DIR/$1
+    echo "Copying diffs to $OUTPUT_DIR/$1"
+    cp rustfmt_diff.txt $OUTPUT_DIR/$1/rustfmt_diff.txt
+    cp feature_diff.txt $OUTPUT_DIR/$1/feature_diff.txt
+
     if [ -z "$diff" ]; then
         echo "no diff detected between rustfmt and the feture branch"
         return 0
     else
+        echo "Copying diffs between rustfmt and feature branch to $OUTPUT_DIR/$1/diff.txt"
+        echo "$diff" >> $OUTPUT_DIR/$1/diff.txt
         echo "$diff"
```

Comment:
@calebcartwright I've updated the code to continue outputting the diff in the logs so it'll be easier to check when we incorporate this in CI.